app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="PrimaryButton" parent="Widget.MaterialComponents.Button">
+        <item name="backgroundTint">@null</item>
+        <item name="android:background">@drawable/bg_primary_button</item>
+        <item name="android:textColor">@color/onboarding_fragment_button_text</item>
+        <item name="android:outlineProvider">none</item>
+        <item name="cornerRadius">@dimen/onboarding_btn_corner_radius</item>
+        <item name="fontFamily">@font/bai_jamjuree_bold</item>
+        <item name="android:textSize">@dimen/onboarding_btn_text</item>
+        <item name="android:height">@dimen/onboarding_btn_height</item>
+        <item name="textAllCaps">false</item>
+        <item name="android:padding">5dp</item>
+    </style>
+
+    <style name="OnboardingTV" parent="Widget.AppCompat.TextView">
```

Comment:
For text styling we go for something generic that matches the style guide and use those in the xml as needed
Heading1, Heading2, Body1 etc

This is because you may the same style in another screen/feature, and you shouldn't have to recreate it each time